playbooks/common-tasks/maas-agent-ubuntu-install.yml

```diff
@@ -46,8 +46,7 @@
 
 - name: Ensure container packages are installed
   apt:
-    pkg: "{{ item }}"
+    pkg: "{{ maas_distro_packages }}"
```

Comment:
Passing a list won't work with Ansible < 2.2 and this set of playbooks tries to cater for 1.9+ so we will have to use a with_items loop.